x-pack/legacy/plugins/siem/public/components/page/hosts/hosts_table/translations.ts

```diff
@@ -7,25 +7,21 @@
 import { i18n } from '@kbn/i18n';
 
 export const HOSTS = i18n.translate('xpack.siem.hostsTable.hostsTitle', {
-  defaultMessage: 'All Hosts',
+  defaultMessage: 'All hosts',
 });
 
 export const UNIT = (totalCount: number) =>
   i18n.translate('xpack.siem.hostsTable.unit', {
     values: { totalCount },
-    defaultMessage: `{totalCount, plural, =1 {Host} other {Hosts}}`,
+    defaultMessage: `{totalCount, plural, =1 {host} other {hosts}}`,
```

Comment:
Nice fixes according to the writing guidelines!

Did a search of `plural, =1` and I see maybe the fields browser could use this lower case treatment as well?

Food for thought, maybe it is different from the others.
<img width="643" alt="Screen Shot 2019-07-31 at 10 53 23 AM" src="https://user-images.githubusercontent.com/1151048/62231610-9f85a600-b381-11e9-97c8-4a2dac6b0a4d.png">

Reply:
👍